webrtc.html

```diff
@@ -1015,29 +1015,48 @@
             <ol>
               <li>
                 <p>If the process to apply <var>description</var> fails for any
-                reason, then user agent MUST queue a task runs the following
+                reason, then user agent MUST queue a task that runs the following
                 steps:</p>
                 <ol>
                   <li>
                     <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
                     <code>true</code>, then abort these steps.</p>
                   </li>
                   <li>
-                    <p>If elements of the SDP were modified, then reject
-                    <var>p</var> with a newly
-                    <a data-link-for="exception" data-lt="create">created</a>
-                    <code>InvalidModificationError</code>
-                    and abort these steps.</p>
-                  </li>
-                  <li>
                     <p>If the <var>description</var>'s <code><a data-for=
                     "RTCSessionDescription">type</a></code> is invalid for the
-                    current <a>signaling state</a> of <var>connection</var>,
+                    current <a>signaling state</a> of <var>connection</var>
+                    as described in <span data-jsep=
+                    "processing-a-local-desc processing-a-remote-desc">[[!JSEP]]</span>,
                     then reject <var>p</var> with a newly
                     <a data-link-for="exception" data-lt="create">created</a>
                     <code>InvalidStateError</code> and abort these steps.</p>
                   </li>
                   <li>
+                    Let <var>lastOffer</var> be the result returned by the
+                    last call to <code>createOffer</code>.
```

Comment:
nit: It's obvious enough to me what this means, but interpreted literally, it's unclear if `lastOffer` is an `RTCSessionDescription`, a `DOMString` or a promise.